frontend/src/components/Home/Home.js

```diff
@@ -5,24 +5,21 @@ import { Nav, Navbar, NavDropdown, Card, Button, Carousel } from 'react-bootstra
 import NavbarToggle from 'react-bootstrap/esm/NavbarToggle';
 import { Container, Col, Row, Form } from "react-bootstrap";
 import './Home.css';
-import {
-  MDBCarousel,
-  MDBCarouselInner,
-  MDBCarouselItem,
-  MDBView,
-  MDBContainer
-} from "mdbreact";
 import logo from '../../../src/images/Stocks_GrowthCombined.png';
+import stocksImage from '../../../src/images/Stocks.jpg';
+import educationalResources from '../../../src/images/EducationalResources.jpg';
+import economicIndicators from '../../../src/images/EconomicIndicators.jpg';
+import news from '../../../src/images/News.jpg';
 // import NavBar from "frontend/src/components/Navbar/Navbar.js";
 
-class Home extends Component {
-    render(){
+function Home() {
+
         return (
             <>
             <Navbar className="color-nav" >
                 
               <Container>
-              <Navbar.Brand href="#home">
+              <Navbar.Brand href="/">
```

Comment:
Either change the width or the height of the navbar brand to auto cause the logo looks a little stretched out. I would recommend letting the width as it is and setting the height to auto.